docs/insomnia/introduction-to-plugins.md

```diff
@@ -53,16 +53,13 @@ The `package.json` configuration includes the following content:
 
 The following is an example minimal `package.json`. The `package.json` must contain an `insomnia` attribute to be identified as an Insomnia plugin. 
 
-```
+```json-doc
```

Comment:
note: this won't work on github (so far as I can see) https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Reply:
Yeah I'm not surprised. It's one of the supported languages for with [rogue](https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers)

Reply:
cool!